packages/components/src/internal/QueryGridModel.ts

```diff
@@ -323,9 +323,14 @@ export class QueryGridModel
         return rows;
     }
 
-    getExportColumnsString(): string {
+    getExportColumnsString(excludeColumns?: string[]): string {
         // does not include required columns -- app only
+        let excludeColumnsLower = [];
```

Comment:
The comment on line 327 really belongs next to the `this.getDisplayColumns` (if anywhere).